editor.planx.uk/src/pages/FlowEditor/lib/analytics/provider.tsx

```diff
@@ -33,11 +33,19 @@ import {
 } from "./utils";
 
 /**
- * If appending to ALLOW_LIST please also update the `analytics_summary` view to
- * extract it into it's own column.
+ * ALLOW_LIST should stay in sync with
+ *   api.planx.uk/modules/webhooks/service/analyzeSessions/operations.ts
+ *
+ * If appending values to ALLOW_LIST please also update the
+ *  `analytics_summary` & `submission_services_summary` views to extract the value into its own column
+ *
+ * Please also ensure your migration ends with `GRANT SELECT ON public.{VIEW_NAME} TO metabase_read_only`
+ *  so that Metabase picks up the new columns
```

Comment:
nit(non-blocking): Maybe these comments point towards the fact we might need a simple doc file explaining how to add values to `ALLOW_LIST`?